pom.xml

```diff
@@ -28,7 +28,7 @@
         <maven.compiler.testSource>1.8</maven.compiler.testSource>
         <maven.compiler.testTarget>1.8</maven.compiler.testTarget>
         <hazelcast.version>1.9.4.8</hazelcast.version>
-        <k3po.version>3.0.0-alpha-34</k3po.version>
+        <k3po.version>develop-SNAPSHOT</k3po.version>
```

Comment:
Will alpha-37 include the changes in [PR#348](https://github.com/k3po/k3po/pull/348)? 

If not, it will not work.